### PR TITLE
models + gem devise added + migrations

### PR DESCRIPTION
**Pull request by SabrinaNayeli**
I added the models made the migrations and added the gem devise. Don't forget to bundle after pulling and before db:migrate !

### CODE REVIEW

**Comment by Tom-Firth** [comment]
Ok, c'est cool !

File: Gemfile.lock [hunk 1]
```diff
@@ -136,6 +144,9 @@ GEM
     rb-inotify (0.9.10)
       ffi (>= 0.5.0, < 2)
     redis (4.0.2)
+    responders (2.4.0)
+      actionpack (>= 4.2.0, < 5.3)
+      railties (>= 4.2.0, < 5.3)
```
Comment (AndyLukac) [inline Gemfile.lock:149]: is this written correctly?

**Comment by ghost** [comment]
Nice !

**Review by AndyLukac — COMMENTED** [review]
Please verify.

File: Gemfile.lock (hunk 1 above)
Comment (Tom-Firth) [reply Gemfile.lock:149]: yes !

**Pull request merged**